test-alerts.cpp

```diff
@@ -4,5 +4,17 @@
 #include "typewise-alert.h"
 
 TEST_CASE("infers the breach according to limits") {
+  //cover inferBreach
   REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
```

Comment:
Test cases can be strengthened to cover more scenarios and hence get good coverage.